pyFAST/input_output/examples/Example_TurbSim_Processing.py

```diff
@@ -0,0 +1,35 @@
+from pyFAST.input_output import FASTInputFile
+from pyFAST.case_generation import runner
+from pyFAST.input_output import TurbSimFile
+import pandas as pd
+
+def main():
+    """Modify TrubSim parameters and write"""
+    filename = '../tests/example_files/FASTIn_TurbSim.inp' # Name of TurbSim's input file
+    f = FASTInputFile(filename)
+    f['WrBLFF'] = False
+    f['WrADFF'] = True
+    f['WrADTWR'] = True
+    f['NumGrid_Z'] = 15
+    f['NumGrid_Y'] = 15
+    f['AnalysisTime'] = 600
+    f.write('../tests/example_files/FASTIn_TurbSim_change.inp')  # Modified file name
+
+    """Run TurbSim"""
+    TurbSim_FILE = '../tests/example_files/FASTIn_TurbSim_change.inp'  # Input file
+    Turbsim_EXE = 'TurbSim_x64.exe'  # Change to the path of the TurbSim executable
+    runner.run_cmd(TurbSim_FILE, Turbsim_EXE, wait=True, showOutputs=False, showCommand=True)
+
+    """Reads the speed of three directions"""

```

Comment:
A more descriptive comment might be: `Open the turbulence box, containing the wind speed in 3 directions"